lib/components/Provider/index.tsx

```diff
@@ -33,8 +34,8 @@ type Props = (
 ) & {
   clientId: string;
   hashedUserId?: string;
-  apiURL?: string;
-  wsURL?: string;
+  apiURL?: API_REGION;
```

Comment:
The type should be both `string` or `API_REGION.` Otherwise, people who upgrade to this version will receive typescript errors. 
Someone who had `apiURL: https://api.ca.notificationapi.com/` should still work without any error.